src/components/Player.vue

```diff
@@ -214,6 +212,7 @@ export default {
 
                     const localPlayer = new this.shaka.Player(videoEl);
                     const proxyURL = new URL(component.video.proxyUrl);
+                    const proxyPath = proxyURL.pathname === "/" ? "" : proxyURL.pathname;
```

Comment:
I think you intend this to be `proxyURL.slice(0, -1)`

Reply:
that will return `""` in case of `/` but `path/with/leading/slash` in case of `/path/with/leading/slash`, what would be wrong

Reply:
But then you add it to URL.pathname, which starts with a /?

You'd end up with a // then

Reply:
but `/` case is covered and `proxyPath` will be an empty string

`/path/with/leading/slash + /url/pathname`
or ` + /url/pathname`

your solution will end with
`path/with/leading/slash + /url/pathname`

Reply:
But let's say `proxyUrl = new URL("https://piped.kavin.rocks/proxy/")`

You'll have problems, here, right?

Reply:
we want to have `uri.pathname = "proxy" + "/url/pathname"`, oki
but what if I have `proxyUrl = new URL("https://piped.kavin.rocks/proxy")`?

can we assume that an `uri.pathname` with no leading slash will build the correct url?

Reply:
will fix ASAP

Reply:
done